odoo/addons/base/tests/test_cli.py

```diff
@@ -14,21 +18,26 @@ def setUpClass(cls):
         super().setUpClass()
         cls.odoo_bin = sys.argv[0]
         assert 'odoo-bin' in cls.odoo_bin
```

Comment:
security asked me to change this bit in my other PR: https://github.com/odoo/odoo/pull/185387#discussion_r1843334491

```diff
-        cls.odoo_bin = sys.argv[0]
-        assert 'odoo-bin' in cls.odoo_bin
+        from os.path import join as opj, realpath  # move me on top
+        cls.odoo_bin = realpath(opj(__file__, '../../../../../odoo-bin'))

Reply:
Ok, but why no
```py
from pathlib import path
cls.odoo_bin = Path(__file__).parents[4].resolve() / 'odoo-bin'
```